core/src/main/java/com/google/instrumentation/stats/MutableView.java

```diff
@@ -19,8 +19,10 @@
 import com.google.instrumentation.stats.ViewDescriptor.DistributionViewDescriptor;
 import com.google.instrumentation.stats.ViewDescriptor.IntervalViewDescriptor;
 import java.util.ArrayList;
+import java.util.HashMap;
```

Comment:
Makes sense. I'll update to keep the order in tests instead of in the implementation.